config/webpack.common.js

```diff
@@ -38,9 +34,10 @@ module.exports = {
         ]
       }
     ],
-    noParse: [/jszip\.js$/]
+    noParse: [/jszip\.js$/, /\.d\.ts$/]
   },
   plugins: [
+    new webpack.IgnorePlugin(/fs/, /file-type/, /debug/), // Ignore fs in music-metadata
```

Comment:
I was hoping that with the introduction of `music-metadata-browser` this stuff, and the relevant polyfills, could be handled inside `music-metadata-browser`'s build process.